e2e/components/UIShell/UIShell-test.e2e.js

```diff
@@ -14,42 +14,26 @@ const { snapshotStory } = require('../../test-utils/storybook');
 test.describe('UIShell', () => {
   themes.forEach((theme) => {
     test.describe(theme, () => {
-      test('header @vrt', async ({ page }) => {
-        await snapshotStory(page, {
-          component: 'UIShell',
-          id: 'components-ui-shell--header',
-          theme,
-        });
-      });
-
-      test('header w/ navigation @vrt', async ({ page }) => {
-        await snapshotStory(page, {
-          component: 'UIShell',
-          id: 'components-ui-shell--header-w-navigation',
-          theme,
-        });
-      });
-
-      test('header w/ actions @vrt', async ({ page }) => {
-        await snapshotStory(page, {
-          component: 'UIShell',
-          id: 'components-ui-shell--header-w-actions',
-          theme,
-        });
-      });
+      // test('header @vrt', async ({ page }) => {
+      //   await snapshotStory(page, {
+      //     component: 'UIShell',
+      //     id: 'components-ui-shell-header--overview',
+      //     theme,
+      //   });
+      // });
```

Comment:
Hey @Gururajj77 should we remove this comments as well?

Reply:
yes, this can be removed.